pkg/test/cpu.go

```diff
@@ -317,7 +317,8 @@ func NoBIOSACMErrors(txtAPI hwapi.LowLevelHardwareInterfaces, config *tools.Conf
 // IA32DebugInterfaceLockedDisabled checks if IA32 debug interface is locked
 func IA32DebugInterfaceLockedDisabled(txtAPI hwapi.LowLevelHardwareInterfaces, config *tools.Configuration) (bool, error, error) {
 	// Check for IA32_DEBUG_INTERFACE support
-	if !cpuid.HasFeature(1 << 11) {
+	_, _, ecx, _ := txtAPI.CPUSignatureFull()
```

Comment:
Function not part of interface definition in go-linux-lowlevel-hw

Reply:
done